packages/credentials-interface/src/status-list-credential.types.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  CredentialStatusPurpose,
+  CredentialStatusType,
+  StatusList2021Entry,
+  VerifiableCredential,
+} from './credentials.types';
+
+export interface StatusList2021 {
+  [key: string]: string;
+  id: string;
+  type: CredentialStatusType.StatusList2021;
+  statusPurpose: CredentialStatusPurpose;
+  encodedList: string;
+}
+
+export interface StatusList2021Credential
+  extends VerifiableCredential<StatusList2021> {}
+
+export const StatusList2021_Context = 'https://w3id.org/vc/status-list/2021/v1';
+
+export const validateStatusListEntry = (
+  statusListEntry: StatusList2021Entry,
+  statusList: StatusList2021Credential
+) => {
+  if (statusListEntry.type !== CredentialStatusType.StatusList2021) {
+    throw new Error('Not a StatusList2021 type');
+  }
+  if (statusListEntry.statusPurpose !== CredentialStatusPurpose.REVOCATION) {
```

Comment:
I would add suspension check also

Reply:
Hmm, may be you right. @jrhender Should this verification fail when credential status is suspended?

Reply:
Yes, I think it should also fail if suspended

Reply:
Ok, thanks